pkg/minishift/addon/parser/addon_parser_test.go

```diff
@@ -51,6 +52,15 @@ var noDescription string = `# Name: One command of each type
 oc foo
 `
 
+var addOnWithMultilineDescription string = `# Name: foo
+# Description: This is one line
+#   This is second line
```

Comment:
Either way this needs to be documented in the docs. If nothing else than a simple TIP in the add-on topic.